Packages/Primitives/Sources/Extensions/BigInt+Primitives.swift

```diff
@@ -28,6 +28,11 @@ public extension BigInt {
         return BigInt(result.rounded())
     }
     
+    func multiply(byPercentage percentage: Double) -> BigInt {
```

Comment:
```suggestion
    func multiply(by value: Double) -> BigInt {
```

so then you call `multiply(by: 0.25)` which is 25% or `multiply(by: 0.5)` 50%